docs/design/datacontracts/datacontracts_design.md

```diff
@@ -34,20 +73,11 @@ Logically a contract may refer to another contract. If it does so, it will typic
 
 ## Types of contracts
 
-There are 3 different types of contracts each representing a different phase of execution of the data contract system.
+There are 2 different types of contracts each representing a different phase of execution of the data contract system.
 
 ### Composition contracts
```

Comment:
These serve similar purposes as the baselines. Is there a rationale for having both Composition contracts and the baselines?

Reply:
Composition contracts are a grouping mechanism for algorithmic contracts.

A logical data descriptor is composed out of physical data descriptors.

It's the similar function but for two different parts of the system.

We still don't have a spec for the physical encoding for contracts in a target runtime. That's not part of the data descriptor stuff in this PR.

Reply:
I removed all references to composition contracts